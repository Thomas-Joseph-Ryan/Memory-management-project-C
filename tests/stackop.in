push a 1
append a 1
pick a 1
pluck a 1
pop a
set c 1
get c
push c 2 3
get c
append c 0 -1
get c
pick c 2
get c
pluck c 2
get c
pop c
get c
pluck c 0