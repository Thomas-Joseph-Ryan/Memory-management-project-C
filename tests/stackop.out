> no such key

> no such key

> no such key

> no such key

> no such key

> ok

> [1]

> ok

> [3 2 1]

> ok

> [3 2 1 0 -1]

> 2

> [3 2 1 0 -1]

> 2

> [3 1 0 -1]

> 3

> [1 0 -1]

> index out of range

> 
bye
